frontend/src/routes/Governance/policies/policy-details/PolicyTemplateDetailsPage.tsx

```diff
@@ -105,27 +107,39 @@ export function PolicyTemplateDetailsPage() {
       return
     }
 
-    const version = apiGroup ? `${apiGroup}/${apiVersion}` : apiVersion
-    fireManagedClusterView(templateClusterName, kind, version, templateName, templateNamespace)
-      .then((viewResponse) => {
-        if (viewResponse?.message) {
-          setTemplateError(viewResponse.message)
-        } else {
-          setTemplate(viewResponse.result)
-        }
-      })
-      .catch((err) => {
-        console.error('Error getting resource: ', err)
-        setTemplateError(err)
-      })
+    let namespace = templateNamespace
+    // Apply both to ValidatingAdmissionPolicy and ValidatingAdmissionPolicyBinding
```

Comment:
We don't support `ValidatingAdmissionPolicy` on this page do we?

Reply:
ValidatingAdmissionPolicyBinding but Users can come here with type URL directly